src/@types/graphql.ts

```diff
@@ -424,7 +424,7 @@ export type ImageAttempt = {
   _id: Scalars['ID']['output'];
   batch?: Maybe<Scalars['String']['output']>;
   created: Scalars['Date']['output'];
-  errors?: Maybe<Array<ImageError>>;
+  errs: Array<ImageError>;
```

Comment:
Making sure this was a purposeful change:`errors` -> `errs`?

Reply:
Ok definitely looks on purpose. Just curious about the reason

Reply:
I did change it back to `errors` in the end.